stubs/mix/tailwind.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  purge: {
+    content: [
+      'source/**/*.blade.php',
+      'source/**/*.md',
+      'source/**/*.html',
+    ]
+  },
+  darkMode: false,
```

Comment:
`false` is the default, right? I can see the value of including the other empty default values here like for `theme` and `variants`, but idk if this one is necessary.

Reply:
Oh yeah sorry I only meant that one line, just `darkMode: false`

Reply:
Oh gotcha! Yeah we can remove that. I just pulled the default stub from Tailwind and configured purge.

Reply:
Agreed, I removed it.